.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Note this was required by storybook for compatibility. See PR description for the output message